packages/sdks/web-component/src/app/index.html

```diff
@@ -32,6 +32,7 @@
         validate-on-blur="false"
         keep-last-authenticated-user-after-logout="true"
         nonce="rAnd0m"
+        dismiss-screen-error-on-input="true"
```

Comment:
do we want it to be the default?

Reply:
It's not the default
It's just for the dev app

Reply:
I meant default in dev-app :)

Reply:
It's not important, we can remove it later